moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp

```diff
@@ -172,27 +173,27 @@ void planning_pipeline::PlanningPipeline::configure()
 void planning_pipeline::PlanningPipeline::displayComputedMotionPlans(bool flag)
 {
   if (display_computed_motion_plans_ && !flag)
-    display_path_publisher_.shutdown();
+    display_path_publisher_.reset();
   else if (!display_computed_motion_plans_ && flag)
-    display_path_publisher_ = nh_.advertise<moveit_msgs::msg::DisplayTrajectory>(DISPLAY_PATH_TOPIC, 10, true);
+    display_path_publisher_ = node_->create_publisher<moveit_msgs::msg::DisplayTrajectory>(DISPLAY_PATH_TOPIC);
```

Comment:
Please use the [QOS history depth parameter](https://github.com/ros2/rclcpp/blob/af9ae4a61c24ba207f3db6a32c41e0e52cb29cb9/rclcpp/include/rclcpp/node.hpp#L165) if not the full [rmw_qos_profile_t](https://github.com/ros2/rclcpp/blob/af9ae4a61c24ba207f3db6a32c41e0e52cb29cb9/rclcpp/include/rclcpp/node.hpp#L182)

Reply:
https://github.com/ros-planning/moveit2/pull/75/commits/bcbceba361836ef7bacb825fdb3784a7339a4a53